packages/arb-token-bridge-ui/src/hooks/useNetworksAndSigners.tsx

```diff
@@ -183,6 +183,25 @@ export function NetworksAndSignersProvider(
     Sentry.setTag('wallet.name', connector?.name ?? '')
   }, [isDisconnected, isConnected, connector])
 
+  useEffect(() => {
+    let l1ChainId, l2ChainId, l1RpcUrl, l2RpcUrl
+    if (result.status === UseNetworksAndSignersStatus.CONNECTED) {
+      l1ChainId = result.l1.network.id
+      l2ChainId = result.l2.network.id
+
+      l1RpcUrl = rpcURLs[l1ChainId]
+      l2RpcUrl = rpcURLs[l2ChainId]
+    }
+
+    Sentry.setTag('network.l1ChainId', l1ChainId ?? '')
```

Comment:
We should lose the notion of L1/L2 because of Orbit chains. Also Sentry uses underscore case so i'd go with these:

```
network.parent_chain_id
network.parent_chain_rpc_url
network.child_chain_id
network.child_chain_rpc_url
```

Reply:
Yeah good point. I've updated to send only these tags now.